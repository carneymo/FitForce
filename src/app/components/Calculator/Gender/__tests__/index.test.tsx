import * as React from 'react';
import { render } from '@testing-library/react';

import { Gender } from '..';

describe('<Gender  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <Gender
        updateGender={function (gender: string): void {
          throw new Error('Function not implemented.');
        }}
      />,
    );
  });
});
