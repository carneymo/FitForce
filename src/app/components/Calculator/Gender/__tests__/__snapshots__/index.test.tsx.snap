// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Gender  /> should match snapshot 1`] = `
<div
  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-19sxs0w-MuiGrid-root"
>
  <div
    aria-label="Gender"
    class="MuiToggleButtonGroup-root css-1gjgmky-MuiToggleButtonGroup-root"
    role="group"
  >
    <button
      aria-pressed="true"
      class="MuiButtonBase-root MuiToggleButton-root Mui-selected MuiToggleButton-sizeSmall MuiToggleButton-primary MuiToggleButtonGroup-grouped MuiToggleButtonGroup-groupedHorizontal css-1dlyelz-MuiButtonBase-root-MuiToggleButton-root"
      tabindex="0"
      type="button"
      value="M"
    >
      Male
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </button>
    <button
      aria-pressed="false"
      class="MuiButtonBase-root MuiToggleButton-root MuiToggleButton-sizeSmall MuiToggleButton-primary MuiToggleButtonGroup-grouped MuiToggleButtonGroup-groupedHorizontal css-1dlyelz-MuiButtonBase-root-MuiToggleButton-root"
      tabindex="0"
      type="button"
      value="F"
    >
      Female
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </button>
  </div>
</div>
`;
